.travis/Dockerfile.debian8.py2

```diff
@@ -0,0 +1,34 @@
+FROM debian:jessie-backports
+
+ENV LANG C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV CI true
+
+RUN echo "deb http://ppa.launchpad.net/lazka/dumpingplace/ubuntu trusty main" >> /etc/apt/sources.list
+
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 5806C7C4
+
+RUN apt-get update
+
+RUN apt-get install -y \
+    quodlibet \
+    exfalso
+
+RUN apt-get install --no-install-recommends -y \
+    python-polib \
+    python-pytest \
+    python-pip \
+    python-faulthandler \
+    python-setuptools \
+    xvfb \
+    gettext \
+    intltool \
+    libxine2 \
+    dbus-x11 
+
+RUN pip install --upgrade pycodestyle pyflakes xvfbwrapper
+
+RUN useradd -ms /bin/bash user
```

Comment:
Some tests failed which used chmod to make a file read only. I thought that since non-root is the usual execution env it can't hurt. Is there any downside?